scarlet/resampling.py

```diff
@@ -0,0 +1,239 @@
+import autograd.numpy as np
+from . import interpolation
+
+
+
+
+def convolve_band(model, psf, padding = 3):
+    """Convolve the model in a single band
+    """
+    psf_fft = np.fft.fft2(np.fft.ifftshift(psf))
+    _model = np.pad(model, padding, 'constant')
+    model_fft = np.fft.fft2(np.fft.ifftshift(_model))
+    convolved_fft = model_fft * psf_fft
+    convolved = np.fft.ifft2(convolved_fft)
+    result = np.fft.fftshift(np.real(convolved))
+  #  (bottom, top), (left, right) = padding
+    result = result#[bottom:-top, left:-right]
+    return result
+
+
+
+def conv2D_fft(shape, xm, ym, p, h, scp):
```

Comment:
This is only used when generating the operator. 
To be honest, my main incentive for using scp here is that it conveniently handles inputs that have different shapes without me having to think of it. :/

Reply:
I'm fine with you using `scipy.signal.fftconvolve` here for now, I just think that you should import the module in `conv2D_fft` instead of passing it as a parameter from another method.